tuxemon/config.py

```diff
@@ -242,6 +248,16 @@ def get_defaults() -> Mapping[str, Any]:
                     )
                 )
             ),
+            (
+                "crash_reporting",
+                OrderedDict(
+                    (
+                        ("log_host_url", "https://transfer.sh/"),
```

Comment:
could we add an option to enable the popup?  during development, its going to crash often, and the popup will get really annoying.

Reply:
I was thinking about adding an option to the config with `None` as the default value (and creating the popup), and bool values for automatically sending/not sending logs.